toxencryptsave/toxencryptsave.c

```diff
@@ -79,7 +79,7 @@ void tox_pass_key_free(Tox_Pass_Key *key)
  *
  * @return true on success.
  */
-bool tox_get_salt(const uint8_t *ciphertext, uint8_t *salt, Tox_Err_Get_Salt *error)
+bool tox_get_salt(const uint8_t *ciphertext, uint8_t salt[TOX_PASS_SALT_LENGTH], Tox_Err_Get_Salt *error)
```

Comment:
Missed `*ciphertext` -> `ciphertext[]`. In fact, you did `ciphertext[TOX_PASS_ENCRYPTION_EXTRA_LENGTH]` in https://github.com/TokTok/c-toxcore/pull/2525/commits/4d4fe66050ac7390e03adf529c27acf2905534e1.

Reply:
Fixed.